nbextensions/bulkImportCell/testAppObj.json

```diff
@@ -228,28 +228,6 @@
             "ui_class": "parameter",
             "ui_name": "Reverse/Right FASTA/FASTQ File Path"
           },
-          {
```

Comment:
Took out this param so Mallory could see the styling with 3 file path params (more similar to what will be shown at the end)